deployment/deploy/6_MocIntegration.ts

```diff
@@ -67,6 +68,7 @@ func.dependencies = [
   "MassetManager",
   "MyntAdminProxy",
   "DeployMockBAssets",
+  "Permit2",
```

Comment:
remove if Permit2 deployment is not needed

Reply:
This is needed